tests/test_subtractTask.py

```diff
@@ -38,7 +38,29 @@
 from lsst.pex.exceptions import InvalidParameterError
 
 
-class AlardLuptonSubtractTest(lsst.utils.tests.TestCase):
+class AlardLuptonSubtractTestBase:
+    def _setup_subtraction(self, doSubtractBackground=False, **kwargs):
+        """Setup and configure the image subtraction PipelineTask.
+
+        Parameters
```

Comment:
`doSubtractBackground = True` is set in `test_background_subtraction`, but that test has to set configs for subtasks which can't be handled by `_setup_subtraction`. I will remove the keyword since it isn't ever used.